static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -5,15 +5,22 @@ import fetch from 'lib/fetch';
 const isCheckbox = element => element.type === 'checkbox';
 const isNamed = element => element.name > '';
 
-const showConfirmation = cForm => {
-    fastdom.write(() => {
-        cForm.textContent = 'Thank you for your contribution';
-    });
+const showConfirmation = () => {
+    const callout = document.querySelector('.element-campaign');
+    if (callout) {
+        fastdom.write(() => {
+            callout.classList.add('success');
+        });
+    }
 };
 
 const showError = cForm => {
+    const errorField = cForm.querySelector('.error_box');
+    const errorPara = document.createElement('p');
+    errorPara.textContent =
+        'Sorry, there was a problem submitting your form. Please try again later.';
     fastdom.write(() => {
-        cForm.append('Sorry, there was an error submitting your contribution');
+        errorField.appendChild(errorPara);
```

Comment:
This will add as many children as there are clicks on the submit button (I had made the same mistake). I would just flat out replace the content like this:
```js
errorField.innerHtml = '<p>Sorry, there was a problem submitting your form. Please try again later.</p>';
```

Reply:
👍